projects/plugins/jetpack/extensions/blocks/button/attributes.js

```diff
@@ -12,6 +12,12 @@ export default {
 	uniqueId: {
 		type: 'string',
 	},
+	customVariant: {
+		type: 'string',
+	},
+	metaName: {
+		type: 'string',
+	},
```

Comment:
Ideally, we would use core button block in future, so I'm not sure if this is just making it harder to migrate? 🤔